image/templates/helm/stackrox-central/values.yaml.htpl

```diff
@@ -538,7 +538,12 @@ central:
 #    # sufficient and manual override is required.
 #    configOverride: null
 #
+[< if .FeatureFlags.ROX_POLICY_AS_CODE >]
+#configAsCode:
+#  enabled: true
+#  installCRDs: true
```

Comment:
Looking at it from a user perspective, I am wondering if we should keep `installCRDs` somewhat internal (for example, for usage by the Operator). I mean, what would be the use-cases where installing the CRDs should be disabled?

Reply:
The current plan is to install the `SecurityPolicy` CRD even when config-as-code is disabled.  The reasoning is that we believe the typical use case for disabling config-as-code will be to turn it off after it was enabled by default as opposed to disabling it up front.  In that use case, we want to keep the CRD installed even when policy-as-code is disabled so that any policy CRs aren't lost.

Given that the CRD will be installed even when policy-as-code is disabled, there needs to be a way for users to disable installing the CRD since it is the only resource that requires elevated privileges and is installed at the cluster scope.

Open to discuss any of the reasoning laid out above.

Reply:
> keep installCRDs somewhat internal

After further consideration, I'd be willing to make this flag more internal.  Can you explain what that means in a bit more concrete terms?